core/controllers/reader.py

```diff
@@ -194,7 +124,7 @@ def classify_string_classifier_rule(state, normalized_answer):
 # TODO(bhenning): Add more tests for classification, such as testing multiple
 # rule specs over multiple answer groups and making sure the best match over all
 # those rules is picked.
-def classify(exp_id, state, answer, params):
+def classify(state, answer):
     """Normalize the answer and select among the answer groups the group in
```

Comment:
Please update this docstring. We are redefining classify to only deal with statistical classification, so everywhere along its call path should be updated accordingly.